golangci.yaml

```diff
@@ -0,0 +1,32 @@
+run:
+  timeout: 6m
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - dogsled
+    - gocritic
+    - goimports
+    - golint
```

Comment:
```suggestion
```

replaced by `revive`:
```
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive.
richgo test -parallel=1 -count=1 -v ./...
```